README.md

```diff
@@ -292,6 +293,14 @@ same major line. Should you need to upgrade to a new major, use an explicit
   project. This means that it will always use the system-wide package manager
   regardless of what is being specified in the project's `packageManager` field.
 
+- `COREPACK_ENV_FILE` can be set to `0` to request Corepack to not attempt to
+  load `.corepack.env`; it can be set to a path to specify a different env file.
+  Only keys that starts with `COREPACK_` will be taken into account, not all
```

Comment:
```suggestion
  Only keys that start with `COREPACK_` will be taken into account, not all
```

Reply:
I would suggest to remove "not all  keys that start with `COREPACK_` will be taken into account" and just put "(however `COREPACK_ENABLE_DOWNLOAD_PROMPT` and `COREPACK_ENV_FILE` are ignored)."

The sentence is otherwise a little difficult to read.

Reply:
I understood the meaning to be:

Only keys that start with `COREPACK_` will be taken into account, except `COREPACK_ENABLE_DOWNLOAD_PROMPT` and `COREPACK_ENV_FILE`, which are ignored.

I may have misunderstood though. If that isn't correct, then perhaps you could explain what is missing?

Reply:
Oh right, I guess it even confused me, proving your point. I wrote it this way because I'm only testing for a few, and I wanted to set expectations that supporting all are not a goal